README.md

```diff
@@ -4,6 +4,12 @@
 
 Stay tuned.
 
+## Arduino set up
+
+For aurdino to be able to access the correct libraries from this repository you will have to change your sketchbook location to the arduino folder cloned from this repository on your local device.
+
+To do this go to *File > Preferences* and change  your sketchbook location on the arduino IDE.
```

Comment:
To what exactly should the sketchbook location be changed?

Reply:
Hi depends on where repository is cloned but should end `/deadcopter/arduino`, will "usually" be something like `/home/{USER}/deadcopter/arduino`

This was to get main branch working at start of the year. We will need to check that libraries used in `due_v2` are in correct folders or we  could run into similar issues

Reply:
When writing instructions on GitHub all paths are with respect to the root path of the project, e.g., `Pi/` or `arduino/due`